README.md

```diff
@@ -79,6 +79,7 @@ Prior to v2.0 this library was known as NTestDataBuilder.
 		    }
 		}
 
+Note that you can optionally override the `BuildObject` method if you want full control over how your object is constructed, for example if you want to use a particular constructor. If you don't provide this then Dossier will create the object for you.
```

Comment:
This sentence is a bit confusing given the example preceeding it has the BuildObject method overridden.

Perhaps something like: "Overriding the `BuildObject` method is optional - if you don't then blagh. It makes sense to override it when blah."